src/pages/timecard/Timecard.spec.jsx

```diff
@@ -318,13 +318,15 @@ describe('Timecard', () => {
         name: 'Previous day',
       });
       expect(previousDayLink.pathname).toBe('/timecard/2022-06-30');
+      fireEvent.click(previousDayLink);
```

Comment:
Is it possible to test that the message summary is cleared when clicking previous/next day?

Reply:
Extra test added